README.md

```diff
@@ -1 +1,29 @@
 # flask_proj05
+
+Install this: https://docs.docker.com/docker-for-mac/install/
```

Comment:
I'm working through all of these comments! Presently working on setting up Docker and figuring out how to run the app using it.